railties/lib/rails/generators/rails/app/templates/.devcontainer/Dockerfile.tt

```
@@ -0,0 +1,14 @@
+FROM mcr.microsoft.com/devcontainers/ruby:1-3-bookworm
```

Shouldn't it be "3-1"?

```suggestion
FROM mcr.microsoft.com/devcontainers/ruby:3-1-bookworm
```

And, I guess, we need to use the current app's Ruby version here.

---

No. The 1 here indicates the version of the image, not the version of ruby.

We plan to host our own images prior to this feature's release, so this is really just a placeholder for now.